go/libraries/doltcore/sqle/sqlfmt/schema_fmt.go

```diff
@@ -23,22 +23,13 @@ import (
 	"github.com/liquidata-inc/dolt/go/libraries/doltcore/schema"
 )
 
-//TODO: This is a relic from before `SHOW CREATE TABLE` was implemented. We should remove this file altogether.
```

Comment:
Is this file still necessary? What's it for?

Reply:
`dolt diff` still uses it to construct statements for diffs. As well, it's used when we auto-generate a schema and want to display it to the user. Can't use the other method since nothing has been written to root yet.

Reply:
The update / insert / deletes can't be handled by the engine, so fine to keep those. But you should figure out how to get all schema serialization to happen in the engine. If not as part of this PR, then as a followup. Having this code lying around when it diverges from the engine implementation is an active liability at this point.